__tests__/app.test.js

```diff
@@ -178,72 +178,128 @@ describe("app", () => {
         .expect(404)
     });
   });
-}); 
 
-describe("GET /api/articles/:article_id/comments", () => {
-  test("Responds with a status of 200 for the right request.", () => {
-    return request(app).get("/api/articles/1/comments").expect(200);
+  describe("GET /api/articles/:article_id/comments", () => {
+    test("Responds with a status of 200 for the right request.", () => {
+      return request(app).get("/api/articles/1/comments").expect(200);
+    });
+    test("200 - Responds with the requested article_id to the client.", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const comments = body.article;
+          expect(Object.keys(comments[0]).length).toBe(6);
+          expect(comments).toHaveLength(11);
+  
+          // Checking the data type
+  
+          comments.forEach((comment) => {
+            expect(typeof comment.comment_id).toBe("number");
+            expect(typeof comment.body).toBe("string");
+            expect(typeof comment.article_id).toBe("number");
+            expect(typeof comment.author).toBe("string");
+            expect(typeof comment.votes).toBe("number");
+            expect(typeof comment.created_at).toBe("string");
+          });
+  
+          // Checking the accuracy of the data for article_id 1
+  
+          expect(comments[0].comment_id).toBe(5);
+          expect(comments[0].body).toBe('I hate streaming noses');
+          expect(comments[0].article_id).toBe(1);
+          expect(comments[0].author).toBe('icellusedkars');
+          expect(comments[0].votes).toBe(0);
+          expect(comments[0].created_at).toBe('2020-11-03T21:00:00.000Z');
+  
```

Comment:
I would check that **every** comment you have has the article_id of 1

Reply:
Done